modules/tinymce/src/plugins/paste/main/ts/core/Utils.ts

```diff
@@ -8,21 +8,27 @@
 import { Unicode } from '@ephox/katamari';
 
 import DomParser from 'tinymce/core/api/html/DomParser';
+import AstNode from 'tinymce/core/api/html/Node';
 import Schema from 'tinymce/core/api/html/Schema';
 import Tools from 'tinymce/core/api/util/Tools';
 
+type RegExpFilter = RegExp | [ RegExp, string ] | [ RegExp, (match: string, ...args: any[]) => string ];
+
+const isRegExp = (val: unknown): val is RegExp =>
+  val.constructor === RegExp;
+
 /**
  * This class contails various utility functions for the paste plugin.
  *
  * @class tinymce.pasteplugin.Utils
  */
 
-const filter = (content, items) => {
+const filter = (content: string, items: RegExpFilter[]): string => {
   Tools.each(items, (v) => {
-    if (v.constructor === RegExp) {
+    if (isRegExp(v)) {
       content = content.replace(v, '');
     } else {
-      content = content.replace(v[0], v[1]);
+      content = content.replace(v[0], v[1] as any);
```

Comment:
This is `as any` as TypeScript can't infer which `replace` overload function to use for the string replace function.